src/test/unit/controllers/PublicationController.test.ts

```diff
@@ -0,0 +1,100 @@
+import PublicationController from '../../../main/controllers/PublicationController';
+import {Response} from 'express';
+import {mockRequest} from '../mocks/mockRequest';
+import sinon from 'sinon';
+import {PublicationService} from '../../../main/service/publicationService';
+import fs from 'fs';
+import path from 'path';
+
+const publicationController = new PublicationController();
+const i18n = {
+  'list-option': {},
+};
+const rawSJPData = fs.readFileSync(path.resolve(__dirname, '../mocks/trimmedSJPCases.json'), 'utf-8');
+const sjpCases = JSON.parse(rawSJPData).results;
+const onePubData = fs.readFileSync(path.resolve(__dirname, '../mocks/onePublication.json'), 'utf-8');
+const onePub = JSON.parse(onePubData);
+describe('Get publications', () => {
+  beforeEach(function() {
+    sinon.stub(PublicationService.prototype, 'getPublications').resolves(sjpCases);
+  });
+  afterEach(function() {
+    sinon.restore();
+  });
+
+  it('should render the Summary of Publications page', async () => {
+
+    const response = {
+      render: () => {
+        return '';
+      },
+    } as unknown as Response;
+
+    const request = mockRequest(i18n);
+    request.query = {courtId: '1'};
+    request.user = {id: 1};
+
+    const responseMock = sinon.mock(response);
+
+    const expectedData = {
+      ...i18n['summary-of-publications'],
+      courtName: '1',
+      publications: sjpCases,
+    };
+
+    responseMock.expects('render').once().withArgs('summary-of-publications', expectedData);
+
+    await publicationController.get(request, response);
+    responseMock.verify();
+  });
+
+  it('should render the SJP if courtId = 0', async () => {
+
+    const response = {
+      render: () => {
+        return '';
+      },
+    } as unknown as Response;
+
+    const request = mockRequest(i18n);
+    request.query = {courtId: '0'};
+    request.user = {id: 1};
+
+    const responseMock = sinon.mock(response);
+
+    const expectedData = {
+      ...i18n['summary-of-publications'],
+      courtName: 'Single Justice Procedure',
+      publications: sjpCases,
+    };
+
+    responseMock.expects('render').once().withArgs('summary-of-publications', expectedData);
+
+    await publicationController.get(request, response);
+    responseMock.verify();
+  });
+});
+
+describe('Get publications2', () => {
+  it('should send data if only one pub is returned', async () => {
+    const response = {
+      send: function () {
+        return '';
+      },
+    } as unknown as Response;
+    const request = mockRequest(i18n);
+    request.query = {courtId: 'x'};
+    request.user = {id: 1};
+    sinon.reset();
```

Comment:
youve already got your reset higher up but it looks like you dont need yo reset your stub, you just need to assign it a var and then return different params when different args are passed
```
(class level) 
const publicationsStub = sinon.stub(PublicationSerivce.prototype, 'getPublications);
publicationsStub.withArgs(CONDTIONA).resolves(solutionA);
publicationStub.withArgs(CONDITIONB).resolves(solutionB);
```
then you wont have to keep resetting stubs